vtkext/private/module/vtkF3DImguiConsole.h

```diff
@@ -43,6 +43,16 @@ class vtkF3DImguiConsole : public vtkF3DConsoleOutputWindow
    */
   void ShowBadge();
 
+  /**
+   * Clear console
+   */
+  void Clear();
+
+  /**
+   * Get console history size
+   */
+  size_t GetLogsSize();
```

Comment:
why is this method needed ?

Reply:
it's not needed, i forgot to remove it after trying to write a test that somehow directly accesses the vtkF3DImguiConsole, sorry 😅